[RUST] ci_cpu docker image build failure.

The ci_cpu docker image fails to build cleanly.  The Dockerfile runs ubuntu_install_rust.sh which in turn executes:
cargo +nightly install rustfmt-nightly --version 1.0.1 --force

Which results in the following build error:

ubuntu_install_rust.sh
^[[0m^[[91m   Compiling scoped-tls v0.1.2
^[[0m^[[91merror[E0277]: the trait bound `std::borrow::Cow<'_, str>: std::convert::From<core::str::EscapeDefault<'_>>` is not satisfied^[[0m^[[91m
^[[0m^[[91m   --> /opt/rust/registry/src/github.com-1ecc6299db9ec823/rustc-ap-graphviz-306.0.0/lib.rs:551:44^[[0m^[[91m
^[[0m^[[91m    |^[[0m^[[91m
^[[0m^[[91m551 |                     (&*s).escape_default().into()^[[0m^[[91m
^[[0m^[[91m    |                                            ^^^^ the trait `std::convert::From<core::str::EscapeDefault<'_>>` is not implemented for `std::borrow::Cow<'_, str>`^[[0m^[[91m
^[[0m^[[91m    |^[[0m^[[91m
^[[0m^[[91m    = help: the following implementations were found:^[[0m^[[91m
^[[0m^[[91m              <std::borrow::Cow<'a, [T]> as std::convert::From<&'a [T]>>^[[0m^[[91m
^[[0m^[[91m              <std::borrow::Cow<'a, [T]> as std::convert::From<&'a std::vec::Vec<T>>>^[[0m^[[91m
^[[0m^[[91m              <std::borrow::Cow<'a, [T]> as std::convert::From<std::vec::Vec<T>>>^[[0m^[[91m
^[[0m^[[91m              <std::borrow::Cow<'a, std::ffi::CStr> as std::convert::From<&'a std::ffi::CStr>>^[[0m^[[91m
^[[0m^[[91m            and 11 others^[[0m^[[91m
^[[0m^[[91m    = note: required because of the requirements on the impl of `std::convert::Into<std::borrow::Cow<'_, str>>` for `core::str::EscapeDefault<'_>`^[[0m^[[91m
^[[0m^[[91m
^[[0m^[[91merror: aborting due to previous error^[[0m^[[91m
^[[0m^[[91m
^[[0m^[[91mFor more information about this error, try `rustc --explain E0277`.^[[0m^[[91m
^[[0m^[[91merror: Could not compile `rustc-ap-graphviz`.
warning: build failed, waiting for other jobs to finish...
^[[0m^[[91merror: failed to compile `rustfmt-nightly v1.0.1`, intermediate artifacts can be found at `/tmp/cargo-installgw3Wp4`

--- a/docker/install/ubuntu_install_rust.sh
+++ b/docker/install/ubuntu_install_rust.sh
@@ -11,11 +11,10 @@ export CARGO_HOME=/opt/rust
 # this rustc is one supported by the installed version of rust-sgx-sdk
 curl -s -S -L https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path --default-toolchain nightly-2019-01-28
 . $CARGO_HOME/env
-rustup toolchain add nightly
 rustup component add rust-src
-cargo +nightly install sccache
-cargo +nightly install rustfmt-nightly --version 1.0.1 --force
-cargo +nightly install xargo
+cargo install sccache
+cargo install rustfmt-nightly --version 1.0.1 --force
+cargo install xargo
 
 # make rust usable by all users
 chmod -R a+w /opt/rust

